lib/util/trip/ride_card.dart

```diff
@@ -92,7 +93,37 @@ class _RideCardState extends TripCardState<Ride, RideCard> {
 
   @override
   Widget buildTopRight() {
-    return Text(key: const Key('price'), ' ${_ride.price}€');
+    if (!_fullyLoaded) {
```

Comment:
This also applies here, and why did we decide to make past rides green depending on whether they are approved? I think they should be grey. If I remember correctly, you can then still tell if a ride is rejected/cancelled because the whole card is grey.

Reply:
past drives that took place are green, past drives taht did not are grey (cancelled)

Reply:
To me it is just to make it more understandable, but yea... you think all grey and greyout the ones that did not take place?

Reply:
Yes I think so too. Makes it more consistent with Drive

Reply:
I would say they are basically rejected from a visual standpoint